app/helpers/automated_tests_helper.rb

```diff
@@ -738,9 +737,53 @@ def self.launch_test(assignment, repo_path, call_on)
     end
   end
 
-  def self.process_result(result, grouping_id, assignment_id)
-  end
+  def self.process_result(raw_result)
+    result = Hash.from_xml(raw_result)
+    repo = @grouping.group.repo
+    @revision = repo.get_latest_revision
+    @revision_number = @revision.revision_number
```

Comment:
I'm a little skeptical that this and revision need to be instance variables... have you thought about this?